### ISSUE TITLE
Want to turn off split screen mode

### ISSUE THREAD

**Issue by internettrollwatt** [microsoft/WSA#232]
### Is your feature request related to a problem? Please describe

At present, WSA automatically turns on the split screen mode, which will bring inconvenience to the experience of some applications (including but not limited to: unable to open the scan QR code interface, and video cannot be played in full screen). I don't know your purpose of turning on the split screen mode, but from the result, it has brought me a bad experience. I hope to solve this problem in the subsequent version. Thank you

### Describe the solution you'd like

Turn off split screen mode by default or add a switch to let the user choose whether to turn on split screen mode

### Please specify the version of Windows Subsystem for Android

2301.40000.4.0

**Comment by nieubank** [comment]
WSA is designed to be a window-first model to best align with customer expectations on Windows devices. This means that we leverage Android's built-in freeform multi-window mode by default. For updating your app to leverage this mode fully please reference [Multi-window support](https://developer.android.com/guide/topics/large-screens/multi-window-support).

You can put an app into full screen mode by pressing F11, we are actively working on making this more discoverable for users in a future release.

**Comment by internettrollwatt** [comment]
Thank you for your reply. I can understand the reason why you turn on split screen mode by default, but it has a great negative impact on me. And after I use F11, this split screen mode has not exited (it has not reached my expected effect). Can you turn off split screen mode for a single application? If you can, I hope to add this function in subsequent versions, thank you

**Comment by nieubank** [comment]
Do you have specific behavioral differences when the window reports as fullscreen vs. freeform? If so, I'd be interested to know why that is and what fullscreen offers that multi-window does not support for you.

We typically see applications making incorrect assumptions when we report a full screen windowing mode as it relates to display layout and resource loading. This causes a poor user experience when the app is in a window and not actually full screen. We explicitly do not want to force users into a full screen experience without user action since launching an app in fullscreen is not typically expected on Windows devices. This is why we encourage developers to adopt the multi-window/freeform mode as part of supporting WSA.

**Comment by internettrollwatt** [comment]
I have installed third-party software through adb. For the protection of personal privacy, I do not want to disclose the names of these software. In short, when using these software to scan QR code or video full-screen playback, these software will prompt "full-screen playback is not possible in split screen mode" or "this function does not support use in split screen mode" (even if I have used F11). Perhaps this is the result of using third-party software, I will feed back this question to the manufacturer of third-party software. Thank you for your reply
